include/GoogleSync/GoogleSync.php

```diff
@@ -790,18 +961,9 @@ public function updateSuitecrmMeetingEvent($event_local, $event_remote)
         $event_local->duration_minutes = $mins;
         $event_local->assigned_user_id = $this->workingUser->id;
 
-        // Disable all popup reminders for the SuiteCRM meeting
-        $sql = "UPDATE reminders SET popup = '0' WHERE related_event_module_id = '" .
-            $event_local->id .
-            "' AND deleted = '0'";
-        $res = $this->db->query($sql);
-
-        // Mark all reminders where both popup and email are disabled as deleted.
-        $sql = "UPDATE reminders SET deleted = '1' " .
-            "WHERE popup = '0' AND email = '0' AND related_event_module_id = '" .
-            $event_local->id .
-            "' AND deleted = '0'";
-        $res = $this->db->query($sql);
+        // Disable all popup reminders for the SuiteCRM meeting. We add them back from Google event below.
+        $event_id = $event_local->id;
+        $this->clearPopups($event_id);
```

Comment:
clearPopups could return false as failure. Please handle the error.

Reply:
handled